src/app/pages/payment/internationalPayment.component.ts

```diff
@@ -0,0 +1,152 @@
+import { Component, OnInit, OnDestroy, ElementRef, ViewChild } from "@angular/core";
+import { InternationalPaymentService } from "../../services/internationalPayment/international-payment.service";
+import { ToastrService } from "ngx-toastr";
+import { Location } from "@angular/common";
+import {
+  FormGroup,
+  FormBuilder,
+  Validators,
+  ReactiveFormsModule,
+  AbstractControl
+} from "@angular/forms";
+import { NgxSpinnerService } from "ngx-spinner";
+import { Router } from "@angular/router";
+import { Subscription } from "rxjs";
+import { SafeUrl, DomSanitizer } from '@angular/platform-browser';
+import { removeSubscription } from 'src/app/helpers/unsubscribe';
+
+@Component({
+  selector: "app-payments",
+  templateUrl: "./internationalPayment.component.html",
+  styleUrls: ["./internationalPayment.component.scss"]
+})
+export class PaymentsComponent implements OnInit {
+  loading: boolean;
+  payload: any;
+  form: FormGroup;
+  formData;
+  subscription: Subscription[] = [];
+  currentUrl: SafeUrl;
+  iframeSrc: SafeUrl;
+  shouldHide: boolean = true;
+  expectedMonths: any = [
+    "01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12"
+  ]
+
+  expectedYears: any;
+
+  constructor(
+    private internationalPaymentService: InternationalPaymentService,
+    private toastrService: ToastrService,
+    private _location: Location,
```

Comment:
Any reason we are preceding the variable **location** with an underscore?

Reply:
Thanks @Victoradukwu done